src/cljs/reagent_toolbox_docs/components/reagent_switch.cljs

```diff
@@ -0,0 +1,128 @@
+;;;; Copyright © 2017 Flexpoint Tech Ltd
+
+(ns reagent-toolbox-docs.components.reagent-switch
```

Comment:
Yeah, I was very much not fond of this either but `switch` is a keyword and instead of fussing down the hole of exclusions, I just went with a different name. It's ugly and I do not like it but it's just semantics. We can sort out how we want to handle this.